chainio/clients/avsregistry/builder.go

```diff
@@ -7,6 +7,48 @@ import (
 	"github.com/Layr-Labs/eigensdk-go/logging"
 )
 
+// Build an AVS registry client with the given configuration,
+// HTTP and WS clients, and logger, but without a private key.
+//
+// This is useful for read-only operations, such as fetching metrics.
```

Comment:
what does it mean by fetching metrics here?

Reply:
Yes, I think can be confusing. I'll remove that bit.

Reply:
Done